procedures/foreman/release.md.erb

```diff
@@ -49,13 +51,15 @@ Note: If for some reason there was an issue with the tarballs that required uplo
 # Packaging a release
 
 [Background documentation](https://projects.theforeman.org/projects/foreman/wiki/Release_Process#Packaging-a-release)
-
+* [ ] Make sure all nightly builds of packages are removed from the releases tags
```

Comment:
Usually we don't build anything nightly in release tags. Did you intend this in the branch docs?

Reply:
Yes I remember that there was a point where I needed to do this, just got the location in the steps wrong.